### ISSUE TITLE
How can I add notes in org beamer mode?

### ISSUE THREAD

**Issue by amosbird** [pdfpc/pdfpc#290]
I'm not sure if it's related to pdfpc though, since I don't know which one provides the noting functionality.

**Comment by AndreasBilke** [comment]
Could you rephrase your question?

**Comment by fnevgeny** [comment]
I guess the requested feature is to automatically output beamer \note's contents to the .pdfpc file - when working under org-mode <http://orgmode.org/worg/exporters/beamer/tutorial.html>. Perhaps <http://cebe.github.io/pdfpc-latex-notes/> might be a starting point, but I really have no clue about internals of org-mode.

More generally, a tighter integration between beamer and pdfpc - not only notes - would be really great.

**Comment by AndreasBilke** [comment]
So the question is org-mode related and has nothing to do with pdfpc,
right?

**Comment by fnevgeny** [comment]
In the present state of affairs, yes, I think so.

**Comment by amosbird** [comment]
yeah, and thanks for letting me know that the notes are in .pdfpc files.
